### PR TITLE
Minor Fixes

### PR DESCRIPTION
**Pull request by Megha-Dev-19**
@Tguntenaar can you please sync your main, it will have other fixes also then

### CODE REVIEW

File: src/devhub/components/templates/AppLayout.jsx [hunk 1]
```diff
@@ -47,13 +58,15 @@ const Footer = (props) => {
...
-        <ContentContainer>{children}</ContentContainer>
-        <Footer page={page} />
-      </Container>
-    </Theme>
+    <ParentContainer>
+      <Theme>
+        <Container className="container-xl">
+          <AppHeader page={page} />
+          <ContentContainer>{children}</ContentContainer>
+          <Footer page={page} />
```
Comment (Tguntenaar) [inline src/devhub/components/templates/AppLayout.jsx:66]: I think the footer needs to be disabled.

**Review by Tguntenaar — CHANGES_REQUESTED** [review]
If you could fix the conflict I can merge and deploy. We should make an github action for that.

**Comment by Megha-Dev-19** [comment]
@Tguntenaar I can't push changes on your main, I don't have the access.

**Comment by Tguntenaar** [comment]
@Megha-Dev-19 you can create a PR to the events-committee branch in the main repo. This branch is the fork

**Pull request closed without merging**